npm/vite-dev-server/index-template.html

```diff
@@ -58,12 +29,6 @@
       Cypress.onSpecWindow(window, importsToLoad)
       Cypress.action('app:window:before:load', window)
 
-      before(() => {
```

Comment:
I am sure we need this - please try running a spec with multiple `it` blocks that mount a React component. One called `react-no-jsx.spec.tsx`.

Reply:
It's possible we don't!

Reply:
Tests in this spec are passing, both in CI and in open mode.

Reply:
Actually scratch that, they only work on open, not in CI...

Reply:
Bizarre

Better fix that before merging, can help out if needed